server/data/services/__mocks__/githubService.js

```diff
@@ -0,0 +1,8 @@
+const create = require('../../../tests/util/create');
```

Comment:
I went and looked at this and I don't think that code warrants its own util module. It can also be rewritten as: 

```js
const create = o => Promise.resolve(o);
```